backend/program/content/overseerr.py

```diff
@@ -1,12 +1,13 @@
 """Mdblist content module"""
+
+from program.indexers.trakt import get_imdbid_from_tmdb
+from program.media.item import MediaItem
+from program.settings.manager import settings_manager
 from utils.logger import logger
 from utils.request import delete, get, ping
-from program.settings.manager import settings_manager
-from program.media.item import MediaItem
-from program.indexers.trakt import get_imdbid_from_tmdb
 
 
-class Overseerr():
+class Overseerr:

```

Comment:
Now that you have so many formatting changes could you mark your formatter and formatting rules?